### DISCUSSION TITLE
The dashboard system information cann't display all apisix nodes.

### DISCUSSION THREAD

**Discussion by liangliwu** [apache/apisix#5928] [General]
The dashboard ver is 2.10.0 . I have deploy three apisix nodes as a cluster.  But the  dashboard system information only display the last boot apisix node. the others node cann't be find in dropdown box。

**Comment by tokers** [comment]
How did you install the other two APISIX nodes? are they deployed with the same way?

**Reply by liangliwu** [reply]
Yes.，the same way deployed. Does the node have a global name configuration?

**Reply by juzhiyuan** [reply]
Hi @nic-chen, please have a look at this issue 😄 Do we have other necessary steps to show all nodes?

**Reply by nic-chen** [reply]
@liangliwu  could you show your configuration for APISIX and the node data stored in etcd?

**Reply by liangliwu** [reply]
etcd : "/apisix/data_plane/server_info/fbee8186-bd40-43e3-9c11-9386ce3ab62b
{"hostname":"t-xxxxx.com.mo","id":"fbee8186-bd40-43e3-9c11-9386ce3ab62b","last_report_time":1640699122,"version":"2.11.0","etcd_version":"3.5.0","up_time":169863,"boot_time":1640529259}

only one node information.


the config.yaml like follow:

apisix:
  node_listen: 9080              # APISIX listening port
  enable_ipv6: false

  allow_admin:                  # http://nginx.org/en/docs/http/ngx_http_access_module.html#allow
    - 0.0.0.0/0              # We need to restrict ip access rules for security. 0.0.0.0/0 is for test.
  ssl:
    ssl_trusted_certificate: /usr/local/apisix/conf/cert/ca.pem

  admin_key:
    - name: "admin"
      key: edd1c9f034335f136f87ad84b625c8f1
      role: admin                 # admin: manage all configuration data
                                  # viewer: only can view configuration data
    - name: "viewer"
      key: 4054f7cf07e344346cd3f287985e76a2
      role: viewer

  enable_control: true
  control:
    ip: "0.0.0.0"
    port: 9092

etcd:
  host:                           # it's possible to define multiple etcd hosts addresses of the same etcd cluster.
    - "https://xxxxx:2379"
    - "https://xxxxx:2379"
    - "https://xxxxx:2379"     # multiple etcd address
  prefix: "/apisix"               # apisix configurations prefix
  timeout: 30                     # 30 seconds
  resync_delay: 5                # when sync failed and a rest is needed, resync after the configured seconds plus 50% random jitter
  health_check_timeout: 10       # etcd retry the unhealthy nodes after the configured seconds
  user: XXXXX            # root username for etcd
  password:  XXXXX       # root password for etcd
  tls:
    cert: /usr/local/apisix/conf/cert/client.pem          # path of certificate used by the etcd client
    key:  /usr/local/apisix/conf/cert/client-key.pem        # path of key used by the etcd client

    verify: true                  # whether to verify the etcd endpoint certificate when setup a TLS connection to etcd,
                                  # the default value is true, e.g. the certificate will be verified strictly.
    sni: "etcd"                   # the SNI for etcd TLS requests. If missed, the host part of the URL will be used.

**Reply by juzhiyuan** [reply]
noted, let's wait for @nic-chen :)

**Reply by nic-chen** [reply]
@liangliwu It should have been fixed by https://github.com/apache/apisix/pull/5558, you could have a try.

**Reply by liangliwu** [reply]
@nic-chen Sorry，I don't understand how to fixed this problem in  #5558

**Reply by juzhiyuan** [reply]
Hi @nic-chen, just to make sure, you mean this issue is fixed and @liangliwu could use the master branch or wait for the next version?

**Reply by liangliwu** [reply]
ok, I have try the master version. but this problem not be fixed yet.

**Reply by liangliwu** [reply]
This problem is solved. sorry, it was my oversight that caused this. I did deploy the apisix nodes in the same way, but when I deployed the other node, I copied the initial node's conf directory to the other node, resulting in duplicate contents of apisix.uid. So the dashboard information can only see the information of one node.

**Reply by juzhiyuan** [reply]
get it, and @liangliwu, do we need to improve documentation so other developers would prevent from this case?

**Reply by liangliwu** [reply]
I think it would be good to have documentation on how the uniqueness of nodes in the cluster is defined. I asked at the beginning of this question if there is a global attribute that defines the node name. I don't see it in the configuration file config.yaml. The online documentation only mentions the configuration of config.yaml and config-default.yaml files.

**Reply by juzhiyuan** [reply]
> how the uniqueness of nodes in the cluster is defined

Hi @tzssangglass, could you please have a look?

**Reply by tzssangglass** [reply]
ref: https://github.com/apache/apisix/blob/master/docs/en/latest/FAQ.md#how-to-customize-the-apache-apisix-instance-id
